### ISSUE TITLE
Cannot build on Android

### ISSUE THREAD

**Issue by signorekai** [fullstackreact/react-native-oauth#44]
Error:
`node_modules/react-native-oauth/android/src/main/java/io/fullstack/oauth/OAuthManagerFragmentController.java:32: error: cannot find symbol
import com.github.scribejava.core.exceptions.OAuthConnectionException;
                                            ^
  symbol:   class OAuthConnectionException
  location: package com.github.scribejava.core.exceptions
/Users/alfredlau/Work/bite/node_modules/react-native-oauth/android/src/main/java/io/fullstack/oauth/OAuthManagerFragmentController.java:281: error: cannot find symbol
      } catch (OAuthConnectionException ex) {`

I'm unable to build my app. Looking into scribejava seems to suggest that exceptions/OAuthConnectionException no longer exists, though I'm not really fluent in Java.

**Comment by tryggvigy** [comment]
Having the same issue. Looking into this.
`OAuthConnectionException` is still present in `scribejava`.
https://github.com/scribejava/scribejava/blob/c9ac8639566a67502fcfca52f3b6aac3d1df96f7/scribejava-core/src/main/java/com/github/scribejava/core/exceptions/OAuthConnectionException.java

But when I run `./gradlew assembleRelease --debug` I get the following:

```
Evaluating project ':react-native-oauth' using build file '/Users/tryggvigylfason/workspace/repos/me/TravAble/app/node_modules/react-native-oauth/android/build.gradle'.
Creating configuration compile.
Creating configuration publish.
Creating configuration provided.
Creating configuration wearApp.
Creating configuration androidTestCompile.
Creating configuration androidTestPublish.
Creating configuration androidTestProvided.
Creating configuration androidTestWearApp.
Creating configuration testCompile.
Creating configuration testPublish.
Creating configuration testProvided.
Creating configuration testWearApp.
Creating configuration debugCompile.
Creating configuration debugPublish.
Creating configuration debugProvided.
Creating configuration debugWearApp.
Creating configuration testDebugCompile.
Creating configuration testDebugPublish.
Creating configuration testDebugProvided.
Creating configuration testDebugWearApp.
Creating configuration releaseCompile.
Creating configuration releasePublish.
Creating configuration releaseProvided.
Creating configuration releaseWearApp.
Creating configuration testReleaseCompile.
Creating configuration testReleasePublish.
Creating configuration testReleaseProvided.
Creating configuration testReleaseWearApp.
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Repo not found or no access token provided: https://jitpack.io/com/github/scribejava/scribejava-apis/maven-metadata.xml]
```

**Comment by BarakChamo** [comment]
Just came across the same issue.

scribejava 4.0 has breaking changes, fixed (for now) by limiting the dependency to
`compile 'com.github.scribejava:scribejava-apis:3.+'` in `build.gradle`

**Comment by tryggvigy** [comment]
Yub, Thanks. modifying `node_modules/react-native-oauth/android/build.gradle` fixed this for me.

Replaced `compile 'com.github.scribejava:scribejava-apis:+'` with `compile 'com.github.scribejava:scribejava-apis:3.+'`

**Comment by tryggvigy** [comment]
Did not see that the latest version has already done this.
https://github.com/fullstackreact/react-native-oauth/commit/93c38f62b9b11ed888a6a6accba980c7ef5fd682

**Comment by auser** [comment]
Yep, the next `npm` release will fix this bug. (Working on #42 currently before I release a new patch version)

**Comment by auser** [comment]
This should be fixed with the latest version (`2.1.14`)
